librz/core/cmd/cmd_print.c

```diff
@@ -7028,10 +7028,10 @@ RZ_IPI RzCmdStatus rz_cmd_print_raw_printable_handler(RzCore *core, int argc, co
 }
 
 RZ_IPI RzCmdStatus rz_cmd_print_raw_string_handler(RzCore *core, int argc, const char **argv) {
-	int len = argc > 1 ? rz_num_math(core->num, argv[1]) : strlen((const char *)core->block);
+	int len = argc > 1 ? rz_num_math(core->num, argv[1]) : core->blocksize;
```

Comment:
i'm not convinced we want to limit a "string-based command" with blocksize. It seems a bit counter intuitive. @wargio WDYT?

Reply:
Agree with @wargio

Reply:
> Would `pr @! some_number` allow a user to specify a max?

yes also. this is why the command also uses the blocksize variable

Reply:
Should I change the test to pr @! 100? @wargio

Reply:
no need